test/features/ticket/data/datasources/ticket_remote_data_source_test.dart

```diff
@@ -28,62 +28,119 @@ void main() {
   });
 
   group('getUserTickets', () {
-    test('should return [Right] when executor returns [Right]', () async {
-      // arrange
-      when(executor.call<List<TicketResponse>>(any))
-          .thenAnswer((_) async => const Right([]));
+    test(
+      'returns a [Right] value '
```

Comment:
i still propose we follow a single format:
"should X if/when Y Z's"

Reply:
We currently don't follow a single format:
```dart
// test/utils/time_since_test.dart
test('time since given midday and 1m30s prior returns a minute ago'), () { ...

// test/data/api/interceptors/authentication_interceptor_test.dart
test('GIVEN a token in SecureStorage WHEN calling onRequest THEN Authorization Header is added to the request', () { ...

// test/data/repositories/v2/account_repository/account_repository_test.dart
test('should call executor', () async { ...

// test/cubits/authentication/authentication_cubit_test.dart
blocTest<AuthenticationCubit, AuthenticationState>(
  'appStarted emits authenticated when a user is stored', ...
```

Although we did try to establish one in `test/README.md`:
```md
...
## Conventions

- A test description must conform to the _GIVEN_, _WHEN_, _THEN_ syntax. Example: _GIVEN no token in
  SecureStorage WHEN calling onRequest THEN no Authorization Header is added to the request_.
...
```

I've updated the names to reflect the syntax written in the README. Any further discussion about the format is out of scope for this PR